ansible/roles/discover/tasks/main.yml

```diff
@@ -0,0 +1,90 @@
+---
+- name: Install discovery-related packages
+  apt:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - avahi-daemon
+    - avahi-utils
+    - libnss-mdns
+    - dnsmasq
+
+- name: Configure avahi-daemon
+  template:
+    src: avahi/avahi-daemon.conf
+    dest: /etc/avahi/avahi-daemon.conf
+
+- name: Create avahi-daemon service
+  template:
+    src: avahi/liquid.service
+    dest: /etc/avahi/services/liquid.service
+
+- name: Don't run dnsmasq as system service
+  service:
+    name: dnsmasq
+    enabled: no
+    state: stopped
+
+- name: Create the /opt/discover folder
+  file:
+    path: /opt/discover
+    state: directory
+    mode: 0755
+
+- name: Download the source code
+  git:
+    repo: https://github.com/liquidinvestigations/discover.git
+    dest: /opt/discover/discover
+
+- name: Create the configuration file
+  template:
+    src: local.py
+    dest: /opt/discover/discover/settings/local.py
+
+- name: Install requirements for discover
+  pip:
+    requirements: /opt/discover/discover/requirements.txt
+    virtualenv: /opt/discover/venv
+    virtualenv_python: python3
+
+- name: Create the libexec folder
+  file:
+    path: /opt/discover/libexec
+    state: directory
+    mode: 0755
+
+- name: Create the runserver script
+  template:
+    src: libexec/runserver
+    dest: /opt/discover/libexec/runserver
+    mode: 755
+
+- name: Create the supervisor conf for discover
+  template:
+    src: supervisor/discover.conf
+    dest: /etc/supervisor/conf.d/discover.conf
+
+- name: Create the supervisor conf for dnsmasq-dns
```

Comment:
Do you really want to move all the configs under `/var/lib/liquid/conf` right now? Excepting the dynamically generated dnsmasq configuration, no configuration is under `/var/lib/liquid`, because that's how the rest of the roles work. 

You want me to move configs for the `discover` role, so everything stops being consistent, or do you want me to move all the configuration files, which should be another patch set and PR?